app/components/SettingsPopover.tsx

```diff
@@ -33,15 +33,20 @@ export const SettingsPopover = () => {
   return (
     <Popover>
       <PopoverTrigger asChild>
-        <IconButton name="settings-fill" variant="pastel" />
+        <IconButton
+          size="sm"
+          name="settings-fill"
+          variant="secondary"
+          className="size-[42px]"
+        />
       </PopoverTrigger>
       <PopoverContent className="max-w-lg px-4">
         <div className={address && 'space-y-3 divide-y divide-outline-base-em'}>
           {address && (
             <div className="space-y-2">
               <ButtonLink
                 size="sm"
-                variant="pastel"
+                variant="secondary"
```

Comment:
At lines 63 and 74 I think we should do the same change we did here, these lines still keep the `variant="pastel"`

Reply:
I think something was off on your diff, cause it's updated